charts/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/hashicorp/consul-k8s/charts
+
+go 1.16
```

Comment:
Is this the minimal version? Should we be using 1.17 now that it's out?

Reply:
I checked our modules, looks like control-plane and cli use 1.16, and acceptance uses 1.14. It should probably be a separate PR to bump them all. I would want to keep them in sync as much as possible too since I think we use specific go versions/images in our CI images

Reply:
Cool